src/ecs/systems/rendering.rs

```diff
@@ -83,10 +90,11 @@ impl<'a, 'b> SystemExt<'a, (&'b EventsLoop, PipelineBuilder, Option<DisplayConfi
 
         world.add_resource(Factory::new());
         world.add_resource(cam);
+        world.add_resource::<Option<AmbientColor>>(None);
```

Comment:
Couldn't we also just add a default ambient light? If somebody doesn't want any, they can always just set it to `[0.0, 0.0, 0.0]`